packages/channel-client/.eslintrc.js

```diff
@@ -1,10 +1,13 @@
+const baseConfig = require('../../.eslintrc.js');
+
 module.exports = {
+  ...baseConfig,
   env: {
     browser: true,
     es6: true
   },
   parser: '@typescript-eslint/parser',
-  plugins: ['@typescript-eslint', 'prettier', 'jest', 'import'],
+  plugins: [...baseConfig.plugins, '@typescript-eslint', 'prettier', 'jest', 'import'],
   extends: [
```

Comment:
You can actually add `../../.eslintrc` to this `extends` field instead of manually extending `module.exports` with `...baseConfig`, I learned.